packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/PickerItemsDefault.scss

```diff
@@ -44,4 +44,5 @@
 
 .itemContainer {
   display: inline-block;
+  vertical-align: top;
```

Comment:
It is in Edge. Without it, items can get out of alignment as you keyboard through them.